assignment.md

```diff
@@ -39,48 +39,40 @@ OVERALL QUESTIONS
 
 ## Git code management commands
 
-<!-- EAJ NOTE: Rearranging because the foundations build on each other -->
-Git includes three commands that you can use to review, update, and publish your changes to and from your local and remote repositories.
-- `fetch` - Retrieve code updates from a remote repository to place into your tracking branch.
-- `pull` - This command retrieves changes from a remote branch to place into your tracking branch before merging them into a local branch.
-- `push` - Send changes from a local branch to a remote repository.
+Git includes three commands that you can use to review, retrieve, and publish changes to and from your local and remote repositories:
+- `git fetch` - Retrieve code updates from a remote repository to place into your tracking branch.
+- `git pull` - This command retrieves changes from a remote branch to place into your tracking branch before merging them into a local branch.
+- `git push` - Send changes from a local branch to a remote repository.
 
 Find more information about these commands in the sections to follow.
 
-### Retrieving code updates (`fetch`)
-The `fetch` command first checks your local branch to determine whether an associated tracking branch exists remotely. If the tracking branch exists, `fetch` retrieves the changes and history from the tracking branch and pulls those changes into the <!-- tracking? remote? --> branch.
+### Retrieving updates from a branch (`git fetch`)
+The `git fetch` command retrieves the changes and history from a tracking branch to a locally-cached version of your remote repository. The `git fetch` command gives you an opportunity to review the changes that others have made to the tracking branch before merging them with the assets in your local branch. You can use `git fetch` to resolve conflicts before they are merged to your local branch.
 
-The `fetch` command does not update your local branch.
+The `git fetch` command does not update your local branch.
 
-<!-- EAJ QUESTIONS:
-    - The text that was originally here was confusing and a jumble of terminology. I am primarily trying to get straight what is updated and where. I understand that it downloads a branch from a remote repository (either the origin or an upstream branch), including the history of each asset. What I don't understand is when we say "retrieve" or "download," where is this downloaded to? It says my local branch isn't updated, but how can I review it locally if my local branch isn't updated? Is it updated on a remote repository?
-    - Does the user need to run fetch before they pull or push? How often?
--->
+To learn more about `git fetch`, review [Git's `git fetch` documentation](https://git-scm.com/docs/git-fetch).
+
```

Comment:
External documentation is super helpful. Because this document is intended to be a fairly brief overview of these three commands without getting into the nitty-gritty, this provides a jumping-off point for users that need it.